drivers/gles3/shaders/scene.glsl

```diff
@@ -8,6 +8,10 @@ mode_additive_instancing = #define USE_ADDITIVE_LIGHTING \n#define USE_INSTANCIN
 mode_depth = #define MODE_RENDER_DEPTH
 mode_depth_instancing = #define MODE_RENDER_DEPTH \n#define USE_INSTANCING
 
+mode_color_multiview = #define BASE_PASS \n#define USE_MULTIVIEW
+mode_additive_multiview = #define USE_ADDITIVE_LIGHTING \n#define MULTIVIEW
+mode_depth_multiview = #define MODE_RENDER_DEPTH \n#define USE_MULTIVIEW
+
```

Comment:
This doubles the number of shaders we need to compile for every shader. I.e. it doubles the shader compilation time for all shaders. So we need to figure out a way to only compile the multiview variant of shaders when multiview is actually needed.

I suggest making multiview a specialization constant that is disabled by default and then only enabled when using multiview shaders.

Reply:
Ok, sounds good! I think using a specialization will make the code in rasterizer_scene_gles3.cpp simpler too

Reply:
Do we have shader specializations in OpenGL? that would indeed simplify this.

Note that we have a project setting that is supposed to enable compiling multiview shaders which by default is off. This is mostly introduced because on Vulkan we compile shaders before they are used and thus before we know whether multiview is required.

I suggest using the same project settings to determine if the multiview specialization is required.